x-pack/plugins/observability/public/components/shared/exploratory_view/hooks/use_add_to_case.ts

```diff
@@ -41,7 +41,11 @@ export const useAddToCase = ({
   lensAttributes,
   getToastText,
   timeRange,
-}: AddToCaseProps & { getToastText: (thaCase: Case | SubCase) => MountPoint<HTMLElement> }) => {
+  appId,
+}: AddToCaseProps & {
+  appId?: 'security' | 'observability';
```

Comment:
Hey @shahzad31 👋  Just curious as to why we need to specify `security` here? 

I'm probably missing something but wouldn't appId always be set to `observability` when it is defined?

cc: @cnasikas

Reply:
I think you are right. The `appId` should be `observability` for you. Also, `AddToCaseProps` contains `appId?: 'security' | 'observability';`